airbyte-integrations/connectors/destination-duckdb/destination_duckdb/destination.py

```diff
@@ -129,9 +167,15 @@ def write(
                     logger.debug(f"Stream {stream_name} was not present in configured streams, skipping")
                     continue
                 # add to buffer
-                buffer[stream_name]["_airbyte_ab_id"].append(str(uuid.uuid4()))
-                buffer[stream_name]["_airbyte_emitted_at"].append(datetime.datetime.now().isoformat())
-                buffer[stream_name]["_airbyte_data"].append(json.dumps(data))
+                logs = {"logs": "test"}
```

Comment:
Looks like this test line should be reverted to remove "test" before merging. Wdyt?

Also, for readability, wdyt of a rename to "record_meta" or similar?

Reply:
Yes, thanks for pointing this out. This needs an implementation according to https://docs.airbyte.com/using-airbyte/core-concepts/typing-deduping#_airbyte_meta-changes.